src/main/java/edu/ucla/library/iiif/fester/CsvParser.java

```diff
@@ -49,13 +49,25 @@
 
     private String[] myCollectionData;
 
+    private String myAVUrlString;
+
     private CsvHeaders myCsvHeaders;
 
     /**
      * Creates a new CsvParser.
+     *
+     * @param aAVUrlString The A/V URL identifier string
      */
-    public CsvParser() {
+    public CsvParser(final String aAVUrlString) {
+        myAVUrlString = aAVUrlString;
+    }
 
+    /**
+     * Creates a new CsvParser.
+     *
+     */
+    public CsvParser() {
+        myAVUrlString = Constants.DEFAULT_AV_STRING;
```

Comment:
I think this constructor should also be removed since it isn't being used

Reply:
I could see the argument for keeping this and the setter instead of the constructor that takes the av string argument. I think the original concern was there was a possibility of a null pointer in case someone used the constructor but didn't set the av string. Here there is a default value set that one can override by using the setter.

Reply:
To make it more generally useful, one might make the default value `http://` or something that would match on everything. In that case, you'd override by using the setter to set your specific A/V server.

Reply:
FTR, in a separate channel we decided to revert to the no-arg constructor and instead pass the A/V URL prefix string to `parse` (reverting my earlier suggestion -- sorry!) which sets us up nicely for rewriting it as a static method